dotCMS/src/main/java/com/dotcms/rest/api/v1/workflow/WorkflowResource.java

```diff
@@ -1262,7 +1263,7 @@ public final Response fireActionByNameMultipart(@Context final HttpServletReques
                                               @Context final HttpServletResponse response,
                                               @QueryParam("inode")            final String inode,
                                               @QueryParam("identifier")       final String identifier,
-                                              @DefaultValue("-1") @QueryParam("language")         final long   language,
+                                              @DefaultValue("-1") @QueryParam("language")         final String   language,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "language" appears 6 times in this file; the first occurrence is on line 1,264](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4324168)